Week3/week3HomeWork/Exercise5/TotalCost.js

```diff
@@ -0,0 +1,20 @@
+const cartForParty = {
+    beers: 1.75, //1.75,
+    chips: 0.99, //0.99,
+    apple: 1.25, //1.25,
+    cake: 0.89, //0.89,
+    wine: 3.15, //3.15
+}
+
+let i;
+
+function calculateTotalPrice(item) {
+    let x = 0;
+    for (i in item) {
+        x += item[i];
```

Comment:
Can you find a better name than `x`? for the variable. Because `x` is not very descriptive

Reply:
okay , i will name it total.